src/types/MediaPlaylist.ts

```diff
@@ -7,8 +7,8 @@ import SegmentTransformer from '../transformers/SegmentTransformer';
 
 export enum MediaPlaylistType {
     Standard = 'STANDARD',
-    IFrame = 'EXT-X-IMAGES-ONLY',
-    Image = 'EXT-X-I-FRAMES-ONLY',
+    IFrame = 'EXT-X-I-FRAMES-ONLY',
+    Image = 'EXT-X-IMAGES-ONLY',
```

Comment:
oof

Reply:
I didn't love it either, but I couldn't think of a better way. I didn't want to make it `MediaPlaylist<T extends SomeType>` because that'd result in massive code changes everywhere we use the library. All we really care about is the convenience getters `isStandardPlaylist`, `isIFramePlaylist` and `isImagePlaylist`